flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/descriptors/ConnectTableDescriptorTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.descriptors;
+
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.internal.Registration;
+import org.apache.flink.table.catalog.CatalogTableImpl;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * Test for {@link ConnectTableDescriptor}.
+ */
+public class ConnectTableDescriptorTest {
```

Comment:
I think we can add a verification in `org.apache.flink.table.client.gateway.local.ExecutionContextTest#testTemporalTables` to check that the `tableEnv.from("EnrichmentSource").getTableSchema` contains the rowtime field. It can cover the bug path. 

It's not clear what the purpose of this test.

Reply:
There is no rowtime field in `tableEnv.from("EnrichmentSource").getSchema`. Rowtime field needs special handle in connector codes when translate `select ...` query.

This test is just for making sure `CatalogTable` should be same to properties.

Reply:
There should be a rowtime field in `tableEnv.from("EnrichmentSource").getSchema`, because `DummyTableSourceFactory` in sql-client handles this.

Reply:
Maybe not.
It is catalog table, not `ConnectorCatalogTable`.
Real invoking is in `StreamTableSourceScan.translateToPlan`, it is hard to validate it.

Reply:
OK, it is from SQL-CLI, it is `ConnectorCatalogTable` now, added test.